api/app/controllers/api/v1/projects/relationships/events_controller.rb

```diff
@@ -10,7 +10,7 @@ class EventsController < ApplicationController
           resourceful! Event, authorize_options: { except: [:index] } do
             Event.filter(
               with_pagination!(event_filter_params),
-              scope: @project.events
+              scope: @project.events.excluding_type(%w(comment_created text_annotated))
```

Comment:
Ensure that there is a spec for this behavior:
EventsController does not return events of the type comment_created or text_annotated

Reply:
✔